index.js

```diff
@@ -56,12 +56,25 @@ function getChromeExe(chromeDirName) {
   return windowsChromeDirectory;
 }
 
+function getChromeDarwin(defaultPath) {
+  if (process.platform !== 'darwin') {
+    return null;
+  }
+
+  if (fs.realpathSync(process.env['HOME'] + defaultPath)) {
```

Comment:
`fs.existsSync` is deprecated, so perhaps we should fix the `getChromeExe` function to use this `fs.realpathSync`.
Let me know if you want me to make this as part of this PR or in a different on.

Thanks.